platforms/ide/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/AbstractClasspathEntry.java

```diff
@@ -188,10 +189,21 @@ public void writeEntryAttributes(Node node) {
 
         Node attributesNode = getAttributesNode(node, "attributes");
 
-        effectiveEntryAttrs.forEach((key, value) ->
-            attributesNode.appendNode("attribute", ImmutableMap.of(
-                "name", key,
-                "value", value)));
+        effectiveEntryAttrs.forEach((key, value) -> {
+            if (value instanceof Iterable) {
```

Comment:
To increase readability, can you please add a comment explaining that we need this support mult-valued classpath entries?

Reply:
Thanks @donat. I added a comment to the added code.

Reply:
Thank you. Let's see if CI has any complaints.